scripts/testing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Git clone the lastest master of Odo
+git clone https://github.com/redhat-developer/odo
+
+# Retrieve the version / what's currently being used as SupervisorD
+IMAGE=`cat odo/pkg/occlient/occlient.go | grep "bootstrapperImage = " | cut -d \" -f2 | sed '/^\s*$/d'`
+
+docker build -t $IMAGE .
+
+rm -rf odo
+
+# Install golang and curl
+sudo apt-get update && sudo apt-get install curl -y
+
+# Curl to /usr/bin/odo
+sudo curl -J -L https://dl.bintray.com/odo/odo/latest/linux-amd64/odo -o /usr/bin/odo
+sudo chmod +x /usr/bin/odo
+
+# Go get Odo
+go get github.com/redhat-developer/odo
```

Comment:
We require this in order to run the e2e integration tests from the $GOPATH.

We are forced to use TravisCI under "bash" mode and to install / get Odo manually rather than "Go" mode since this repo isn't exactly a Go project, but rather a Docker image repo.

If that makes any sense!

Reply:
Ah yes, I see :) 
Have had similar issues with CircleCI.

One thing I am not clear about though (being a go n00b), when we do `go get github.com/redhat-developer/odo` do we pull master or the latest released version?

Reply:
Yup. `go get` gets the latest source code.

I use bintray to get the binary so we don't have to spend time building the binary each-time within this test code, but now that you bring it up, I may just do that instead..

Reply:
I understand the rationale, what you have done makes perfect sense :)
I just want to be sure that the source and binary match :P